packages/e2e-tests/specs/sidebar.test.js

```diff
@@ -124,12 +124,14 @@ describe( 'Sidebar', () => {
 			removeEditorPanel( 'featured-image' );
```

Comment:
It would be great to add a check a few lines earlier whether `Status & Visibility` is there before it gets removed:

```js
expect( await findSidebarPanelWithTitle( 'Status & Visibility' ) ).toBeDefined();
```